Make users aware of traefik problems (#4036)
rancher/docs: content/k3s/latest/en/installation/network-options/_index.md

> Dual-stack is supported on k3s v1.21 or above.

Reviewer: ```suggestion
Dual-stack is supported on K3s v1.21 or above.
```

> To enable dual-stack in k3s, you must provide valid dual-stack `cluster-cidr` and `service-cidr` on all server nodes. Both servers and agents must provide valid dual-stack `node-ip` settings. Note that [Traefik does not correctly handle ingress resources on a dual-stack cluster](https://github.com/traefik/traefik/issues/8406). Besides, only vxlan backend is supported at the moment. This is an example of a valid configuration:

Reviewer: ```suggestion
To enable dual-stack in K3s, you must provide valid dual-stack `cluster-cidr` and `service-cidr` on all server nodes. Both servers and agents must provide valid dual-stack `node-ip` settings. 

Note that Traefik does not correctly handle ingress resources on a dual-stack cluster. As such, we recommend that users disable Traefik on K3s if they want dual-stack and instead use ingress-nginx. In addition, only vxlan backend is supported currently. The following is an example of a valid configuration:
```

Reviewer: We try not to reference GH issues as such in the docs; rather, we try and describe the ongoing issue and workaround for the users. We can add such issues to existing or upcoming release notes if desired/applicable.

> IPv6 only setup is supported on k3s v1.22 or above. As in dual-stack operation, IPv6 node addresses cannot be auto-detected; all nodes must have an explicitly configured IPv6 `node-ip`. This is an example of a valid configuration:

Reviewer: ```suggestion
IPv6 only setup is supported on K3s v1.22 or above. As in dual-stack operation, IPv6 node addresses cannot be auto-detected; all nodes must have an explicitly configured IPv6 `node-ip`. This is an example of a valid configuration:
```
